netctl/netctl.go

```diff
@@ -252,6 +252,71 @@ func listNetworks(ctx *cli.Context) {
 	}
 }
 
+func createTenant(ctx *cli.Context) {
+	argCheck(1, ctx)
+
+	tenant := ctx.Args()[0]
+	subnetPool := ctx.String("subnet-pool")
+	subnetLen := ctx.Int("subnet-len")
+	vlans := ctx.String("vlans")
+	vxlans := ctx.String("vxlans")
+
+	if subnetPool == "" || vlans == "" || vxlans == "" {
```

Comment:
I think that this should be handled as part of objmodel as well, else the check has to be repeated across all netctl CLI (like create network which uses subnet as well). From what I remember @shaleman was mentioning he was working on sanitizing some of these like making sure the subnets don't overlap across tenants. I'm guessing this will be taken care as part of that.